doc/version_history.rst

```diff
@@ -6,6 +6,11 @@
 Version History
 ###############
 
+v1.3.2
+======
+
+* Update Jenkinsfile.conda to Jenkins Shared Library
```

Comment:
Please add `* Pin the versions of ts_idl and ts_salobj in conda/meta.yaml.` or similar